### ISSUE TITLE
Migrated Chair and Members not displaying affiliation

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/laevigata#1734]
**ISSUE**
When I view a migrated ETD like https://staging-upgrade-etd.emory.edu/concern/etds/db78tc90h ( see [legacy version](
https://legacy-etd.library.emory.edu/view/record/pid/emory:s29v9) ) committee members affiliations are not displayed.  When I edit them, names that should be Emory affiliated are listed as "Non-Emory", but with no institution listed.

Server: https://staging-upgrade-etd.emory.edu
Version v2.0.0-beta7 updated 21 August 2018

**STEPS TO REPRODUCE**
1. Login to staging-upgrade as an administrator
1. Find any migrated ETD - e.g. db78tc90h
1. Scroll down to the Committee Chair and Committee Members listing
1. Click on edit
1. Scroll down to the Committee section

**EXPECTED BEHAVIOR**
* All three faculty should be listed as Emory affiliated in the show and edit view
    * [Hockenberry, Jason Michael](https://www.sph.emory.edu/faculty/profile/#!JHOCKEN)
    * [Cummings, Janet](https://www.sph.emory.edu/faculty/profile/#!JRCUMMI)
    * [Pitts, Stephen R](http://hip.emory.edu/resources/faculty/pitts_stephen.html)

**ACTUAL BEHAVIOR**
* No affiliation is shown in the show view
* All three faculty are shown as "Non Emory" in the edit view

SHOW VIEW
=======
<img width="458" alt="1734 - committee affiliations not displaying" src="https://user-images.githubusercontent.com/3064318/44434809-b79a5600-a572-11e8-97e7-502496b38dde.png">

EDIT VIEW
========
<img width="450" alt="1734 - committee affiliations incorrect in edit view" src="https://user-images.githubusercontent.com/3064318/44434815-bff29100-a572-11e8-8737-2856ca8bc1d1.png">

**Comment by bess** [comment]
Nope,  it's not just a re-indexing problem. The data really isn't there:
```
irb(main):006:0> etd.committee_chair.first.name_and_affiliation
=> "Hockenberry, Jason Michael"
irb(main):008:0> etd.committee_chair.last.name_and_affiliation
=> "Cummings, Janet"
irb(main):009:0> etd.committee_chair.first.affiliation
=> []
```

@no-reply Would you please take a look at whether this data was present in the record pre-migration and didn't make it over? Or does this data just not exist in the Fedora 3 object?

**Comment by mark-dce** [comment]
Confirmed fixed in v2.0.0
![1734 - Committee Member Affiliation - FIXED.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/55118693805715190031f073/125516c66e82c728ace21e0d46b9c3c977c2cee9ace6b24ead49e4912c060a762406a63930a4e52eb64e72001b0611f251454f328591fc5fb4aa2e30c80351b882294d7ed43819f8eb8a19b854bd76df1cd1b1bc8a4d91bc3a23b383d95d220c4749d7ab0268f9d31e8a4afc59e04a5fa5163566e27030754f901a7d2b792a.png)
